devtools/create-gossipstore.c

```diff
@@ -4,24 +4,44 @@
 #include <ccan/read_write_all/read_write_all.h>
 #include <common/type_to_string.h>
 #include <common/utils.h>
+#include <devtools/create-gossipstore.h>
 #include <fcntl.h>
 #include <gossipd/gen_gossip_store.h>
 #include <gossipd/gossip_store.h>
 #include <inttypes.h>
 #include <stdio.h>
+#include <stdlib.h>
 #include <sys/stat.h>
 #include <sys/types.h>
 #include <unistd.h>
 #include <wire/gen_peer_wire.h>
 
+
+
+struct scidsat * load_scid_file(FILE * scidfd) 
```

Comment:
trivial: spacing after `*` is unusual.
Also, functions should be `static` by default; the only exception is when functions need to be accessed by other C files.